addon/components/sl-grid.js

```diff
@@ -665,7 +669,14 @@ export default Ember.Component.extend({
         'content.length',
         'totalCount',
         function() {
```

Comment:
Does this computed property need to return `null`?  Seems `Boolean` is the only valid return value?